pipelines/skylab/slideseq/SlideSeq.changelog.md

```diff
@@ -1,8 +1,14 @@
-# 3.0.0
+# 2.1.8
```

Comment:
version number should probably by 3.0.1, right?

Reply:
I guess it depends on how we want to increase the version number. I think it was increased from 2.1.6 to 3.0.0 in the loom to h5ad PR. I just readjusted the version number for that change to 2.1.7 and then increased the one here to 2.1.8.

Reply:
I talked to the team about it, and we updated to 3.0.0 b/c a slideseq output was removed. I set it back to 3.0.0 and 3.0.1